src/app/components/Search/index.tsx

```diff
@@ -113,7 +127,20 @@ const SearchCmp: FC<SearchProps> = ({ scope, variant, disabled, onFocusChange: o
   const onFormSubmit = (e?: FormEvent) => {
     e?.preventDefault()
     if (value) {
-      navigate(RouteUtils.getSearchRoute(scope, value))
+      let wantedValue = value.trim()
+      let underSpell = false
```

Comment:
I don't love the variable names. Something like `isUnderSpellNext` would better differentiate it from `isUnderSpell`

Reply:
OK, cleaned up